elytron/src/main/java/org/wildfly/extension/elytron/SSLDefinitions.java

```diff
@@ -282,9 +283,8 @@
             .setAllowExpression(false)
             .build();
 
-    private static final SimpleAttributeDefinition MAXIMUM_CERT_PATH = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.MAXIMUM_CERT_PATH, ModelType.INT, true)
+    static final SimpleAttributeDefinition MAXIMUM_CERT_PATH = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.MAXIMUM_CERT_PATH, ModelType.INT, true)
             .setAllowExpression(true)
-            .setDefaultValue(new ModelNode(5))
```

Comment:
Why this change?

Reply:
this is required to retain backward compatibility, as the older configuration has max-cert-path defined inside CRL and now it should be in trust-manager. If this still had default value, there would be no way of knowing if user defined it or if it is the default. (Now you can have max-cert-path either in CRL or trust-manager, but not both)

Reply:
Okay, I see now. There is a bit pity that with this change, user does not know what does it mean when it is not configured at all - does it mean unlimited? If this default value is really not possible to set in the code, we should update attribute description at least, though.

Another approach might be to have these attributes defined as separate - e.g.: MAXIMUM_CERT_PATH and MAXIMUM_CERT_PATH_CRL? With this change, we could also mark MAXIMUM_CERT_PATH_CRL attribute definition as deprecated in the CLI which is something we should do anyway I think. Not sure how much additional complexity this approach brings in.

Reply:
It is set in code to 5, but not in the model, it is later when the service starts and both MAXIMUM_CERT_PATH for TM and CRL are undefined. But I will split them to make the CRL one deprecated